docs/docs/recipes.md

```diff
@@ -119,6 +119,7 @@ In Gatsby, you access data through a query language called [GraphQL](https://gra
 - Read through a conceptual guide on [querying data with GraphQL in Gatsby](/docs/querying-with-graphql/)
 - Learn more about the `graphql` tag -- [querying data in a Gatsby page](/docs/page-query/)
 - Learn more about `<StaticQuery />` -- [querying data in (non-page) components](/docs/static-query/)
+- Learn about [querying data in non-page components](/docs/querying-data-components)
```

Comment:
I would add here also the StaticQuery and useStaticQuery keywords

Reply:
@muescha what do you mean by keywords? Also, that change is going away if you read through the PR.

Reply:
```suggestion
- Learn about [Use StaticQuery and useStaticQuery hook in non-page components](/docs/querying-data-components)
```

Reply:
if there was a link then the full title should show up in the link text

Reply:
but this change is going away so i can resolve the conversation